subjects/devops/kept-promise/README.md

```diff
@@ -0,0 +1,56 @@
+## kept-promise
+
+### Instructions
+
+Things in your code do not always happen as fast as you would like.
+
+Sometimes it is important to wait for a function to finish before keep going on. Other times you don't want your code to be blocking while waiting.
+
+Create an asynchronous function `processInfo` in a file `kept-promise.js`. The function `processInfo` will take an asynchronous function as an input, and it will print the message `Ok!` if the return of the input function is zero or a multiple of two, `Error!` otherwise.
+
+The following function can be used as an example of input function for `processInfo`:
+
+```js
+const getImportantInfo = async _ =>
```

Comment:
I know that it works, but that doesn't mean is correct, unless you need for some reason to declare a parameter that is not going to be used, it would be better to use `()`